moonraker_obico/nozzlecam.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import time
+from moonraker_obico.webcam_capture import capture_jpeg
+_logger = logging.getLogger('obico.nozzlecam')
+
+class NozzleCamConfig:
+    def __init__(self, snapshot_url):
+        self.snapshot_url = snapshot_url
+        self.snapshot_ssl_validation = False
+
+class NozzleCam:
+
+    def __init__(self, app_model, server_conn):
+        self.config = app_model.config
```

Comment:
Is this used anywhere? Did you add it here and later on changed the design and forgot to remove it? It seems that you often make basic mistakes like this. When programing, keep 2 things in mind:

1. Do things that are easy to forget first.
2. Do things in an order that, if you forget something, the program will yell at you.